content/en/news/kubesphere-2021-year-in-review.md

```diff
@@ -0,0 +1,119 @@
+|Riyao Gao|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-gaoriyao.png) |
+|Lijie Jiang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-jianglijie.png)|
+|Fei Xu|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-xufei.png) |
+|Lei Qiao|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-qiaolei.png) |
+|Ning Qi|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-qining.png) |
+|Xiaofan Luan|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-luanxiaofan.png) |
+|Zongwei Tan|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-tanzongwei.png) |
+|Zhengjun Zhou|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-zhouzhengjun.png) |
+|Ju Lou|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-louju.png) |
+|Chao Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-zhangchao.png) |
+|Weifeng Sheng|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-shengweifeng.png) |
+|Haili Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-zhanghaili.png) |
+|Yuchuan He|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-heyuchuan.png) |
+|Tao Hu|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-hutao.png) |
+|Zhenfei Pei|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-peizhenfei.png) |
+|Sheng Zou|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-zousheng.png) |
+|Pei Zhou|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-zhoupei.png) |
+|Changqing Wu|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-wuchangqing.png) |
+|He Huang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-huanghe.png) |
+|Yang Lin|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-linyang.png) |
+|Siwei Gu|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-gusiwei.png) |
+|Xiaomeng Zhang|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-zhangxiaomeng.png) |
+|Dehan Liu|[Download Certificate](https://kubesphere-community.pek3b.qingstor.com/images/speaker-liudehan.png) |
+
+## Looking Ahead
+
+2021 had finally drawn to an end. we are so lucky and grateful to all of our innovative users, customers, and developers, who helped us to push our boundaries in developing an enterprise-grad Kubernetes platform and keeping fast iteration. It is impossible to complete these milestones without your effort and engagement. 
```

Comment:
`we are` -> `We are`

`enterprise-grad` -> `enterprise-grade`

`effort and engagement` -> `efforts and engagements` (This is a suggestion.)